src/app/main/main.component.ts

```diff
@@ -51,6 +51,9 @@ export class MainComponent implements AfterViewInit {
     this.ngxSmartModalService.getModal('classicModal').onAnyCloseEventFinished.subscribe((modal: NgxSmartModalComponent) => {
       console.log('All close events are ended!', modal);
     });
+
+    // Needed to avoid "Expression has changed since the last check cycle..."
+    this.cdr.detectChanges();
```

Comment:
Normally, the user doesn't have to do that since the library will be compiled, but in dev it's not excluded that he has to put this into `AfterviewInit` if he has `*ngIf` conditions on `.hasData()` in template.